rfcs/protocols/data-sharing/fabric.md

```diff
@@ -8,7 +8,50 @@
 - RFC: 02-002
 - Authors: Venkatraman Ramakrishna, Sandeep Nishad, Krishnasuri Narayanam, Dhinakaran Vinayagamurthy
 - Status: Proposed
-- Since: 01-Dec-2020
+- Since: 02-Jan-2023
 
 ## Summary
 
+This document specifies the Hyperledger Fabric implementation of modules, and application adaptation guidelines, for the asset transfer protocol.
+Within Weaver, the protocol units to operate on the asset being transferred will be implemented in the Fabric Interoperation Chaincode, which is the [interoperation module](../../models/infrastructure/interoperation-modules.md) for a Fabric-based network.
+- The functions are meant to be called only by relay drivers with special wallet identities that are recognized by the Fabric Interoperation Chaincode at bootstrap.
```

Comment:
If its just one point in bullet, would a new para be better?

Reply:
I made the entire section a bullet list. Also made a similar change in the Fabric asset transfer RFC.